helloextend-protection/includes/class-helloextend-global.php

```
@@ -405,28 +408,25 @@ public function helloextend_init_global()
-        }
+		if ($store_id){
+			if ($helloextend_enabled === '1') {
+				wp_enqueue_script('helloextend_script');
+				wp_enqueue_script('helloextend_global_script');
+				wp_localize_script('helloextend_global_script', 'ExtendWooCommerce', compact('store_id', 'ajaxurl', 'environment'));
+
+				// Get the leadToken from URL parameters
+				$lead_token = $this->get_lead_token_from_url();
+				if ($lead_token) {
+					// Sanitize the token for safe JavaScript output
+					$safe_lead_token = esc_js($lead_token);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$safe_lead_token'. (undefined)

(UnusedLocalVariable)